syspurpose/src/syspurpose/files.py

```diff
@@ -261,17 +257,14 @@ class SyncedStore(object):
     PATH = USER_SYSPURPOSE
     CACHE_PATH = CACHED_SYSPURPOSE
 
-    def __init__(self, uep, on_changed=None, consumer_uuid=None, report=None, use_valid_fields=False):
+    def __init__(self, uep, on_changed=None, consumer_uuid=None, use_valid_fields=False):
         self.uep = uep
         self.filename = self.PATH.split('/')[-1]
         self.path = self.PATH
         self.cache_path = self.CACHE_PATH
-        self.report = report
```

Comment:
I think removing this will break the current implementation of "subscription-manager {usage|addons|role} --show".
This could be fixed by removing the use of reports from subscription_manager/syspurposelib.py.

I'm also curious why remove the report? Granted it's only used for debug logging but it seemed to me to provide potentially useful logging in debugging issues in syncing syspurpose.

Reply:
You are right, I broke it :open_mouth:

Reply:
Why? I wanted to clean the code a little. The `report` was something out of `syspurpose` package and using it only for accessing protected attributes didn't look like as good approach.